lp-builder-config/ceph.yaml

```diff
@@ -126,6 +126,16 @@ projects:
     charmhub: ceph-fs
     launchpad: charm-ceph-fs
     repository: https://opendev.org/openstack/charm-ceph-fs.git
+    branches:
+      master:
+        build-channels:
+          charmcraft: "2.0/stable"
+        channels:
+          - latest/edge
+        bases:
+          - "20.04"
+          - "22.04"
+          - "22.10"
```

Comment:
Although slightly counter-intuitive, this is only going to configure ceph-fs with one branch: master.  i.e. configuration of `branches` here *overwrites* all of the default set of branches, not just the single one mentioned.  If it does need to be different, just for the master branch, then the whole set of default branches needs to be copied here, and then the master one updated.

Reply:
I noticed there's a default section in lp-builder-config/ceph.yaml. By any chance would that handle the other branches? If not I'll add the other branches.

Reply:
Sorry, read your comment closer and you mentioned the default set. Let me add the branches.